### PR TITLE
Asserts snap build docs style

### PR DESCRIPTION
**Pull request by emgee**
Update & improve SnapBuild doc comments (based on reviews while adding SnapRevision).

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

File: asserts/snap_asserts.go [hunk 1]
```diff
@@ -24,35 +24,36 @@ import (
...
 }
 
-// SnapID returns the snap id of the built snap.
+// SnapID returns the snap id of the snap.
 func (snapdcl *SnapBuild) SnapID() string {
 	return snapdcl.Header("snap-id")
 }
 
-// SnapDigest returns the digest of the built snap.
+// SnapDigest returns the digest of the snap submitted to the store. The digest
```
Comment (pedronis) [inline asserts/snap_asserts.go]: I would leave out "submitted to the store" here

**Comment by pedronis** [comment]
ok to test

File: asserts/snap_asserts.go (hunk 1 above)
Comment (emgee) [inline asserts/snap_asserts.go]: Good point, removed.

**Comment by elopio** [comment]
@pedronis you are not m-o admin right now, only chipaca, mvo5, niemeyer. 
https://github.com/ubuntu-core/snappy-jenkins/blob/master/containers/jenkins-master/config/jobs/github-snappy-integration-tests-cloud/config.xml#L48
It's easy to add you, but I'm not sure what was the plan @fgimenez had here. Federico, wouldn't it be good to enable allowMembersOfWhitelistedOrgsAsAdmin?

File: asserts/snap_asserts.go [hunk 2]
```diff
@@ -24,35 +24,36 @@ import (
...
+// SnapBuild holds a snap-build assertion, asserting the properties of a snap
+// at the time it was built by the developer.
 type SnapBuild struct {
 	assertionBase
 	size      uint64
 	timestamp time.Time
 }
 
-// SnapID returns the snap id of the built snap.
+// SnapID returns the snap id of the snap.
```
Comment (elopio) [inline asserts/snap_asserts.go]: Would it be ok like this? "SnapID returns the id of the snap."

**Comment by elopio** [comment]
Thanks, I find the docs clearer now. :+1: I'll leave the Merge button to pedronis.

**Pull request merged**